modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/file/FilePageStoreManager.java

```diff
@@ -1058,6 +1056,8 @@ public static String cacheGroupName(File dir) {
             return name.substring(CACHE_GRP_DIR_PREFIX.length());
         else if (name.startsWith(CACHE_DIR_PREFIX))
             return name.substring(CACHE_DIR_PREFIX.length());
+        else if (name.startsWith(MetaStorage.META_STORAGE_DIR_NAME))
```

Comment:
``startWith`` can be replaced to ``equals``

Reply:
Fixed.